Dockerfile

```diff
@@ -39,14 +35,4 @@ RUN git checkout 7912389334e347bbb2eac0520c836830875c39de
 RUN make
 RUN go install
 
-WORKDIR $GOPATH
-RUN go install github.com/xlab/c-for-go@master
-RUN mkdir -p src/github.com/renproject
-WORKDIR $GOPATH/src/github.com/renproject
-RUN git clone https://github.com/renproject/solana-ffi
-WORKDIR $GOPATH/src/github.com/renproject/solana-ffi
-RUN git checkout 720c0143d8655bfcd412ae73c949474df2c1dcf3
-RUN make clean && make
-RUN go install ./...
-
-RUN git config --global --remove-section url."https://${GITHUB_TOKEN}:x-oauth-basic@github.com/"
\ No newline at end of file
+COPY ./solana-ffi $GOPATH/src/github.com/renproject/solana-ffi
```

Comment:
It might be best to keep the build step in the dockerfile itself to prevent cross-platform issues with local build

Reply:
The reason I move this step out of the dockerfile is to try to prevent github token leak. 
The previous approach will have the github token stored in the image.

Reply:
Because the `make` command for solana-ffi needs to access some private repo, I wasn't able to find a good way to securely pass the token during the building process, so I decided to move that to the CI and just copy the result files.

Reply:
would it be if we do multi stage builds? Like we build the solana stuff in the first stage and then copy over the data to the second stage? As far as I remember final stage data is all that remains in the image

Reply:
we can also tryu docker build secrets seems like it was created for this specific purpose https://pythonspeed.com/articles/docker-build-secrets/